cmd/kubeapps/migrate.go

```diff
@@ -84,4 +89,5 @@ var migrateCmd = &cobra.Command{
 
 func init() {
 	RootCmd.AddCommand(migrateCmd)
+	migrateCmd.Flags().String("dst-tiller-namespace", "kube-system", "Namespace of target Tiller.")
```

Comment:
`dst` sounds weird to me, maybe it should be `target` since that's the description, even though it's a bit longer?